src/components/Modals/MobileWelcome/views/Notice.tsx

```diff
@@ -17,6 +19,11 @@ export const Notice = () => {
   const handleRecoveryClick = () => {
```

Comment:
Not required for this PR but as this was touched, can we `useCallback()` while at it?